.github/workflows/actionlint.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           # SC2086 - Double quote to prevent globbing and word splitting. - https://www.shellcheck.net/wiki/SC2086
           SHELLCHECK_OPTS: --exclude=SC2046,SC2086
         with:
-          fail_on_error: true
+          fail_level: error
```

Comment:
https://github.com/reviewdog/action-actionlint/commit/9543997a0f5033286cfa134e35118b8701f352a7, `fail_on_error` is deprecated. Should I move this into a separate PR?

Reply:
I'm fine with the current approach